src/pages/my_page/user_profile/ProfilePictureUploader.tsx

```diff
@@ -0,0 +1,147 @@
+import AddAPhotoIcon from '@mui/icons-material/AddAPhoto';
+import CancelIcon from '@mui/icons-material/Cancel';
+import { Box, Button, CircularProgress, Skeleton, Typography } from '@mui/material';
+import { useMutation, useQuery } from '@tanstack/react-query';
+import { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { useDispatch } from 'react-redux';
+import { fetchProfilePicture, uploadProfilePicture } from '../../../api/cristinApi';
+import { PageSpinner } from '../../../components/PageSpinner';
+import { setNotification } from '../../../redux/notificationSlice';
+import { dataTestId } from '../../../utils/dataTestIds';
+
+interface ProfilePictureUploaderProps {
+  id: string;
+}
+
+export const ProfilePictureUploader = ({ id }: ProfilePictureUploaderProps) => {
+  const { t } = useTranslation();
+  const dispatch = useDispatch();
+  const [isLoading, setIsLoading] = useState(false);
+
+  const profilePictureQuery = useQuery({
+    queryKey: ['picture', id],
+    queryFn: () => fetchProfilePicture(id),
+    meta: { errorMessage: false },
+    retry: false,
+  });
+
+  const mutateProfilePicture = useMutation({
+    mutationFn: (base64String: string) => uploadProfilePicture(id, base64String),
+    onSuccess: () => {
+      dispatch(setNotification({ message: t('feedback.success.upload_profile_photo'), variant: 'success' }));
```

Comment:
Denne brukes for både å legge til- og fjerne profilbilde, så da burde kanskje både nøkkel og oversettelsen endres til noe som passer begge flytene? Feks `update_profile_photo` og `Profilbildet ble oppdatert`